EIPS/eip-6969.md

```diff
@@ -0,0 +1,169 @@
+---
+eip: 6969
+title: Contract Secured Revenue on an EVM based L2
+status: Draft
+type: Core
+discussions-to: https://t.me/+w1EKtAx824VlZDJh
+author: Zak Cole <zak@numbergroup.xyz>, Kevin Owocki <kevin@supermodular.xyz>
+created: 2023-05-01
+---
+
+## Abstract
+
+Contract Secured Revenue (CSR) allows smart contract developers to claim a percentage of all transaction fees paid by users when interacting with their smart contracts.
+
+This EIP proposes the introduction of CSR on EVM-based L2s. If adopted, this would allow smart contract developers who deploy on L2s to access a potentially lucrative revenue stream and/or fund public goods.
```

Comment:
You want to write as if the proposal has been adopted, because once it goes to final, it won't make sense to say "if adopted".